frontend/tests/e2e/apply/features/happy_path_apply.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Happy Path – Apply Workflow
+  As a grant applicant
+  I want to complete the full Apply workflow successfully
+  So that I can submit a complete application without errors
+
+  Background:
+    Given the user is able to login with all roles needed
+    And the system is available
+
+  @Happy-Path
+  Scenario: Complete the full Apply workflow successfully
+
+    # --- Login ---
+    Given the user launches the URL
+    When the user enters valid login credentials
+    And the user logs in
+    Then the user goes to the search page
+    When the user searches for the specific Funding Opportunity number
+    Then the user is taken to the Funding Opportunities page
+
+    # --- Starting Application ---
+    When the user clicks "Start Application"
+    Then the "Start a new application" modal opens
+    When the user selects a specific option in the "Who's applying" drop down
+    And enters the name of the application
+    And clicks on "Create Application" button
+    Then a new workspace is created
+    And the Apply page loads with navigation and header
+
+    # --- Completing Required and Optional Forms ---
+    Given there is one specific required form "SF-424"
+    When the user completes all required fields in the SF-424 form
+    And clicks on "Save"
+    Then the system saves the SF-424 form
+
+    Given there are conditionally required forms
+    When the user completes all required fields in one of the optional forms
+    And clicks on "Save"
+    And selects "Yes" for the optional form filled
+    And ensures remaining optional forms have "No" selected
+    Then the system saves the optional form
+    And the user is taken to the next required form or step
+
+    # --- Save Behavior ---
+    When the user clicks "Save" in the form
+    Then no validation errors should appear
+
+    # --- Uploading Documents ---
+    When the user uploads all required documents under the "Attachment" header
```

Comment:
Attachments and documents, somewhat confusingly, are required by individual forms rather than by the application or the competition. The attachment section of the application page is, at this point, only a place to see which attachments have been uploaded across all forms. I think this section could maybe be something more like:
```
When the user uploads an attachment to a form
Then the attachment appears in the application page Attachments section
```